torch/onnx/symbolic_opset10.py

```diff
@@ -20,6 +20,44 @@
 # release on 04/24/19
 
 
+def div(g, self, other, *args):
```

Comment:
The ONNX changes need a test, too. See here:

https://github.com/pytorch/pytorch/blob/fcf8b712348f21634044a5d76a69a59727756357/test/onnx/test_pytorch_onnx_onnxruntime.py#L1308

The test does not need to be extensive, it just needs to validate that the call works correctly.